src/js/users/components/CreateUserForm.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import { useForm } from "react-hook-form";
+import {
+    Checkbox,
+    InputError,
+    InputGroup,
+    InputLabel,
+    InputSimple,
+    ModalBody,
+    ModalFooter,
+    SaveButton,
+} from "../../base";
+
+interface CreateUserFormProps {
+    /** The user's handle or username */
+    handle?: string;
+    /** The user's password */
+    password?: string;
+    /** Error message to be displayed */
+    error: string;
+    /** A callback function to be called when the form is submitted */
+    onSubmit: (data: { handle: string; password: string; forceReset: boolean }) => void;
+}
+
+/**
+ * A form component for creating a new user
+ */
+export function CreateUserForm({ handle = "", password = "", error, onSubmit }: CreateUserFormProps) {
+    const [forceReset, setForceReset] = useState(false);
+
+    const {
+        formState: { errors },
+        register,
+        handleSubmit,
+    } = useForm({ defaultValues: { handle, password, forceReset } });
+
+    function handleForceReset() {
+        forceReset ? setForceReset(false) : setForceReset(true);
```

Comment:
Nitpick:
```suggestion
       setForceReset(!forceReset);
```
 Also I would probably just attach this an an anonymous function